check_logs.py

```diff
@@ -962,6 +962,16 @@ def print_testing_farm_result(args, result):
     print("")
 
 
+def sanitize_for_actions(text):
+    safe_lines = []
+    for line in text.splitlines():
+        # if it starts with :: we prepend a space so it can’t be parsed as a command
+        if line.startswith("::"):
+            line = " " + line
+        safe_lines.append(line)
+    return "\n".join(safe_lines)
```

Comment:
Not sure if that's actually faster than `re.sub(r"^::", " ::", flags=re.M)`, but re would certainly be simpler. But bikeshedding/matter of taste.